crates/macro-support/src/parser.rs

```diff
@@ -411,7 +411,7 @@ impl<'a> ConvertToAst<BindgenAttrs> for &'a mut syn::ItemStruct {
             };
 
             let attrs = BindgenAttrs::find(&mut field.attrs)?;
-            assert_not_variadic(&attrs)?;
+            // assert_not_variadic(&attrs)?;
```

Comment:
Why have dead code in there? Can you please add a comment explaining why this is commented out?

Reply:
Removed, no longer needed.